server/app/routes/products/index.js

```diff
@@ -28,8 +40,11 @@ router.get('/', function (req, res, next) {
         .catch(next);
 });
 
-router.post('/', function(req, res, next){
+router.post('/:userId', function(req, res, next){
```

Comment:
I think this could be a security problem if we don't use req.user.id unfortunately.